libpius/state.py

```diff
@@ -1,73 +1,74 @@
+        ])
 
-  def __iter__(self):
-    return self.state.__iter__()
+    def __iter__(self):
+        return self.state.__iter__()
 
-  def signed(self, key):
-    return key in self.state and self.state[key] == SignState.kSIGNED
+    def signed(self, key):
+        return key in self.state and self.state[key] == SignState.kSIGNED
 
-  def will_not_sign(self, key):
-    return key in self.state and self.state[key] == SignState.kWILL_NOT_SIGN
+    def will_not_sign(self, key):
+        return key in self.state and self.state[key] == SignState.kWILL_NOT_SIGN
 
-  def update(self, key, val):
-    # we don't store NOT_SIGNED, it's meaningless.
-    if val == SignState.kNOT_SIGNED:
-      return
-    self._validate_value(val)
-    self.state[key] = val
-    self.modified = True
+    def update(self, key, val):
+        # we don't store NOT_SIGNED, it's meaningless.
+        if val == SignState.kNOT_SIGNED:
+            return
+        self._validate_value(val)
+        self.state[key] = val
+        self.modified = True
 
-  def save(self):
-    SignState.store_signed_keys(self.state)
+    def save(self):
+        SignState.store_signed_keys(self.state)
 
-  @classmethod
-  def load_signed_keys(self):
-    if not os.path.exists(SignState.kPIUS_SIGNED_KEYS):
-      return dict()
-    fp = open(SignState.kPIUS_SIGNED_KEYS, 'r')
-    data = fp.read()
-    try:
-      signstate = json.loads(data)
-    except:
-      signstate = dict((key, 'SIGNED') for key in data.strip().split("\n"))
-    fp.close()
-    return signstate
+    @classmethod
+    def load_signed_keys(cls):
+        if not os.path.exists(SignState.kPIUS_SIGNED_KEYS):
+            return dict()
+        with open(SignState.kPIUS_SIGNED_KEYS, 'r') as fp:
+            data = fp.read()
+            try:
+                signstate = json.loads(data)
+            except:
+                signstate = {key: 'SIGNED' for key in data.strip().split("\n")}
+        return signstate
 
-  @classmethod
-  def store_signed_keys(self, signstate):
-    # re-read in the list and merge it...
-    prev_signstate = SignState.load_signed_keys()
-    # merge the two with the one we're passed in winning
-    result = dict(prev_signstate.items() + signstate.items())
-    if not os.path.exists(PIUS_HOME):
-      os.mkdir(PIUS_HOME, 0750)
-    if not os.path.isdir(PIUS_HOME):
-      print ('WARNING: There is a ~/.pius which is not a directory.'
-             ' Not storing state.')
-      return
-    fp = open(SignState.kPIUS_SIGNED_KEYS, 'w')
-    fp.write(json.dumps(result))
-    fp.close()
+    @classmethod
+    def store_signed_keys(cls, signstate):
+        # re-read in the list and merge it...
+        prev_signstate = SignState.load_signed_keys()
+        # merge the two with the one we're passed in winning
+        result = dict(prev_signstate.items() + signstate.items())
+        if not os.path.exists(PIUS_HOME):
+            os.mkdir(PIUS_HOME, 0o750)
```

Comment:
If anything it increases readability and keeps the same convention as the other non-decimal types.

`0b` - binary
`0o` - octal
`0x` - hexadecimal